images/ci-kubernetes-e2e-kubeadm/Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = 0.4
+VERSION = 0.5
```

Comment:
I thought that this pattern (`x.y`) was the standard in test-infra? Is there an issue for migrating all of these images to use a different versioning scheme?